nuxeo-features/nuxeo-elasticsearch/nuxeo-elasticsearch-core/src/test/resources/elasticsearch-test-nested-contrib.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0"?>
+<component name="org.nuxeo.elasticsearch.test.override.hints.contrib">
```

Comment:
```suggestion
<component name="org.nuxeo.elasticsearch.test.nested.hints.contrib">
```
To avoid collision with the other `elasticsearch-test-override-hints-contrib.xml`.